.github/workflows/build_test.yml

```
@@ -58,16 +58,23 @@ jobs:
         uses: actions/checkout@v1
         with:
           submodules: recursive
-
+          
+      - name: setup environment
+        run: bash ./scripts/dev_setup.sh -b -t -m
+        
+      - name: Install Rust (stable)
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)